.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 bower_components/
 config/config.js
 rise-financial-version.js
+rise-financial.html
```

Comment:
@donnapep I don't think we should ignore these files. When a designer uses Bower or npm to pull in this component, those files will be missing. They need to be able to use an ES5 version.